package.json

```diff
@@ -2,7 +2,7 @@
   "name": "animated",
   "version": "0.2.0",
   "description": "Declarative Animations Library for React and React Native",
-  "main": "lib/index.js",
+  "main": "lib/targets/react-dom",
```

Comment:
Why this change? Would `react-native` and `react-native-web` I need to import from `animated/lib/targets/react-native` to get this working?